### ISSUE TITLE
Panic Occurred

### ISSUE THREAD

**Issue by costelabr17** [Atmosphere-NX/Atmosphere#1508]
[](url)Title ID: 524850534F53454D
ERROR: KRNL ( 0xF00 )

this error shows when i try boot atmosphere using fss0 as setup, if i use fusee-primary as payload, console only show black screen, 
sysnand 11.0.1 
![WhatsApp Image 2021-05-15 at 11 05 31](https://user-images.githubusercontent.com/43475497/118364389-9ec9fe00-b56e-11eb-9e23-1ecf740294c0.jpeg)
[https://www.mediafire.com/file/otc90f6tkzxo84e/report_01d8a747.bin/file](url)

**Comment by SciresM** [comment]
Screenshot of your atmosphere/contents folder?

The error occurs due to invalid INI binary in memory when kernel is doing setup.

**Comment by costelabr17** [comment]
i put atmosphere 0.19.3 release without any sysmodule the inject fusee-primary now show my this error and generate this report [https://www.mediafire.com/file/e3jtj9tnvdd1lvf/report_0000000001d7260d.bin/file](url)
![Uploading WhatsApp Image 2021-05-15 at 16.25.23.jpeg…]()

**Comment by SciresM** [comment]
You did not post a screenshot of the atmosphere/contents folder, which you were requested to do.

Do you happen to have some kind of mod adding extra RAM or something crazy to your switch?

My best guess is that or that your installed package2 is malformed.

**Comment by SciresM** [comment]
Didn't mean to close this yet.

**Comment by costelabr17** [comment]
sorry i will send it on monday, i forgot my console at work, tks for the help

**Comment by costelabr17** [comment]
![atmo](https://user-images.githubusercontent.com/43475497/118535988-89e19c00-b721-11eb-92ec-eb999df45d32.png)

**Comment by Erionn** [comment]
I'm having a similar issue, but with a different error code.  I updated from 12.0.1 to 12.0.2 last night and Atmosphere 19.3, had no issues until I rebooted this morning and started getting this:

![IMG_20210527_122047](https://user-images.githubusercontent.com/35086507/119911641-895db800-bf0e-11eb-92e2-bb43f42361d9.jpg)

I've tried a fresh install of atmosphere, I have nothing else on the SD card other than what is necessary for it to boot. I can get into hekate just fine, OFW boots fine, but I get that same error any time I try to boot into atsmophere, whether it's through hekate or just through injecting fusee-primary.
